freqtrade/persistence.py

```diff
@@ -231,13 +240,16 @@ def adjust_stop_loss(self, current_price: float, stoploss: float, initial: bool
         if not self.stop_loss:
             logger.debug("assigning new stop loss")
             self.stop_loss = new_loss
+            self.stop_loss_pct = stoploss
             self.initial_stop_loss = new_loss
+            self.initial_stop_loss_pct = stoploss
```

Comment:
Please add tests to `test_adjust_stop_loss` for the 2 new fields... checking if one remains, the other moves, ...

I'm also not sure on the sign ... you store it "as is" ... but the calculation in line 237 supports positive and negative stoplosses - so maybe we should do an `-1 * abs(stoploss)` to match that behavior (stoploss should always be negative - according to constants.py anyway ...